chapters/hpmor-chapter-000.tex

```diff
@@ -34,9 +34,9 @@ \section*{Something, somewhere, somewhen, must have happened differently…}
 % \end{center}
 
 \section*{Some notes}
-The opinions of characters in this story are not necessarily those of the author. What warm!Harry thinks is \emph{often} meant as a good pattern to follow, especially if Harry thinks about how he can cite scientific studies to back up a particular principle. But not everything Harry does or thinks is a good idea. That wouldn't work as a story. And the less warm characters may sometimes have valuable lessons to offer, but those lessons may also be dangerously double-edged.
+The opinions of characters in this story are not necessarily those of the author. What warm!Harry thinks is \emph{often} meant as a good pattern to follow, especially if Harry thinks about how he can cite scientific studies to back up a particular principle. But not everything Harry does or thinks is a good idea. That wouldn’s work as a story. And the less warm characters may sometimes have valuable lessons to offer, but those lessons may also be dangerously double-edged.
```

Comment:
"wouldn't" got changed to "wouldn’s"!

Reply:
thanks, fixed as well in https://github.com/rrthomas/hpmor/pull/83